docs/basics.md

```diff
@@ -367,7 +367,7 @@ To use a different port for the healthcheck:
 
 ### Servers
 
-Servers are simply defined using a `URL`. You can also apply a custom `weight` to each server (this will be used by load-balancing).
+Servers are simply defined using a `url`. Note that paths in `url` are not used when sending traffic to the servers; use `Modifer` to specify paths. You can also apply a custom `weight` to each server (this will be used by load-balancing).
```

Comment:
I just added `!!! note` block, and kept the previous lines for minimal changes.

For other changes (like wrapping lines, removing trailing white space) I think it would be better to have separate PR.